pkg/algorithms/algorithmFactory.py

```diff
@@ -12,7 +12,7 @@
 class AlgorithmFactory: # pylint: disable= too-few-public-methods, too-many-arguments, line-too-long
     """Algorithm Factory to choose algorithm
     """
-    def instantiate_algorithm(self, algorithm: str, matcher: Matcher, dataframe:pd.DataFrame, test: dict, options: dict, metrics_config: dict[str,dict]):
+    def instantiate_algorithm(self, algorithm: str, matcher: Matcher, dataframe:pd.DataFrame, test: dict, options: dict, metrics_config: dict[str,dict], version_field: str = "ocpVersion"):
```

Comment:
is this a rebase issue or did you make these changes?

Reply:
Had to make the changes, as I think orion by default expects the field for the ocp version to be ocpVersion, whereas we name it as cluster_version in our data